bundles/org.openhab.binding.amplipi/src/main/java/org/openhab/binding/amplipi/internal/AmpliPiGroupHandler.java

```diff
@@ -159,10 +164,12 @@ public void receive(Status status) {
     private void updateGroupState(Group state) {
         this.groupState = state;
 
+        Boolean power = !groupState.getMute();
```

Comment:
Just a question to asses if this is intended. As it is now, power is allways the inverted value of mute. The readme shows  "Whether the zone/group is active or off", is it right that it is not a switch to power on/off but just to group (un)mute?

Reply:
Yes, technically, power is nothing else than muting a zone/group.
I nonetheless wanted to add this channel as a solution to the problem described in https://github.com/openhab/openhab-core/pull/3990.